flink-libraries/flink-table/pom.xml

```diff
@@ -126,14 +125,15 @@ under the License.
 			<plugin>
 				<groupId>net.alchim31.maven</groupId>
 				<artifactId>scala-maven-plugin</artifactId>
-				<version>3.1.4</version>
+				<version>3.2.2</version>
```

Comment:
Why did you update the plugin version?
The problem is now that we are using different plugin versions across the project.
I would suggest to remove the plugin versions from all usages of the plugin and put it into the `pluginManagement` section of the root-pom.